### ISSUE TITLE
Add alt to images

### ISSUE THREAD

**Issue by sadhikahuria** [sadhikahuria/sadhikahuria#10]
For better usualbility, as well as accessibility, all the images should have alt text. Some might be missing. Check and add alt text to all.

**Comment by sadhikahuria** [comment]
The background colour of the button merges in with the other colours of the content
